.travis.yml

```diff
@@ -1,6 +1,6 @@
 sudo: required
 language: node_js
-node_js: '8'
```

Comment:
Should not we run tests with our real target?

Reply:
What do you mean?

Reply:
It is just a temporary commit, see the commit message and the [TODO list](https://github.com/theia-ide/theia/pull/3358#issue-227359223).